pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -54,6 +55,13 @@ type NodeConfig struct {
 	Client *client.Client
 	// A client to connect to Docker
 	DockerClient dockertools.DockerInterface
+
```

Comment:
I think there's enough shared now to warrant including the options (configapi.NodeConfig) directly.